background.js

```diff
@@ -21,18 +21,8 @@ async function sendMessage({ message }) {
     return await messenger({ tabId: activeTab.id, message })
 }
 
-async function registerScripts() {
-    await chrome.scripting.registerContentScripts([{
-        id: 'script-content',
-        js: ['content-script.js'],
-        runAt: 'document_idle',
-        matches: ['*://*.youtube.com/*']
-    }])
-}
-
 
```

Comment:
Mind removing the extra line here? There's currently two lines acting as a separator between code blocks instead of 1.